stdlib/universe/difference.go

```diff
@@ -18,6 +18,7 @@ type DifferenceOpSpec struct {
 	NonNegative bool     `json:"nonNegative"`
 	Columns     []string `json:"columns"`
 	KeepFirst   bool     `json:"keepFirst"`
+	InitialZero bool     `json:"initialZero"`
```

Comment:
Are we going to have this value be a boolean or a customizable value?

Reply:
The boolean value would be sufficient if we just want to give an initial value to zero. 
Wondering if giving some other initial value will make sense in our use case?